register/templates/registration/email.html

```diff
@@ -1,9 +1,10 @@
 {% extends "registration/registration_base.html" %}
 {% load i18n %}
 
-{% block title %}{% trans "Enter your email" %}{% endblock %}
+{% block title %}{% trans "Login or create an account" %}{% endblock %}
 
 {% block content %}
+<h1>Login or create an account</h1>
```

Comment:
I'd make the string as translatable, even if the i18n workflow is not yet finalized.